src/components/step-sequence/step-sequence-item/step-sequence-item.component.tsx

```diff
@@ -1,29 +1,53 @@
 import React, { useContext } from "react";
-import PropTypes from "prop-types";
-import StepSequenceItemStyle from "./step-sequence-item.style";
-import StepSequenceItemContentStyle from "./step-sequence-item-content.style";
-import StepSequenceItemIndicatorStyle from "./step-sequence-item-indicator.style";
-import StepSequenceItemHiddenLabelStyle from "./step-sequence-item-hidden-label.style";
+import {
+  StepSequenceItemStyle,
+  StepSequenceItemContentStyle,
+  StepSequenceItemIndicatorStyle,
+  StepSequenceItemHiddenLabelStyle,
+} from "./step-sequence-item.style";
 import Icon from "../../icon";
 import { StepSequenceContext } from "../step-sequence.component";
 
-const StepSequenceItem = (props) => {
+export interface StepSequenceItemProps {
+  /** Aria label */
+  ariaLabel?: string;
+  /** Hidden label to be displayed if item is complete */
+  hiddenCompleteLabel?: string;
+  /** Hidden label to be displayed if item is current */
+  hiddenCurrentLabel?: string;
+  /** Value to be displayed before text for uncomplete steps */
+  indicator: string;
+  /** Flag to hide the indicator for uncomplete steps */
+  hideIndicator?: boolean;
+  /** Status for the step */
+  status?: "complete" | "current" | "incomplete";
+  /** Content to be displayed */
+  children: React.ReactNode;
+}
+
+export const StepSequenceItem = ({
+  hideIndicator = false,
+  indicator,
+  status = "incomplete",
+  hiddenCompleteLabel,
+  hiddenCurrentLabel,
+  children,
+  ...rest
```

Comment:
according to the type definition, `rest` here can only contain `ariaLabel`. If that's the intention, I'd rather be explicit about it by skipping `...rest` and just mentioning the `ariaLabel` prop explicitly here.